rollup.config.js

```diff
@@ -31,7 +31,7 @@ export default [
     external: ["react", "react-dom", "styled-components"],
   },
   {
-    input: "build/esm/types/index.d.ts",
+    input: "build/esm/index.d.ts",
```

Comment:
I'm confused why this was ever working. I don't think `"build/esm/types/index.d.ts"` has ever existed.